vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailActivity.kt

```diff
@@ -114,6 +121,13 @@ class RoomDetailActivity :
             }
         }
         views.drawerLayout.addDrawerListener(drawerListener)
+
+        playbackTracker.trackActivity(ROOM_DETAILS_SCREEN_ON_TRACKER) { isActive ->
```

Comment:
@bmarty using a different approach, this version is checking for all audio playback/recording activity within the `RoomDetailActivity`.

The previous version was only taking into account the voice recording view/composer, the new version also includes timeline audio playback